libpcan/CMakeLists.txt

```diff
@@ -1,35 +1,32 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libpcan)
 
-find_package(catkin REQUIRED COMPONENTS rosbuild roslib mk rosbuild rospack)
+find_package(catkin REQUIRED COMPONENTS mk rospack)
```

Comment:
No `DEPENDS` for `mk` and `rospack`, as not part of an exported interface.
But I'll still `find_package` it, so that `catkin_lint` does not complain...